pkg/fanal/analyzer/analyzer.go

```diff
@@ -444,6 +445,10 @@ func (ag AnalyzerGroup) AnalyzeFile(ctx context.Context, wg *sync.WaitGroup, lim
 				Options:  opts,
 			})
 			if err != nil && !errors.Is(err, aos.AnalyzeOSError) {
+				if ignoreErrors(err, opts.IgnoreErrors) {
+					log.Logger.Debugf("Ignore error for %s", filePath)
+					return
+				}
 				log.Logger.Debugf("Analysis error: %s", err)
 				return
```

Comment:
We don't return errors here. 
I am not sure - perhaps we want to return errors after adding `ignore-errors` flag.